install.sh

```diff
@@ -1,4 +1,12 @@
 set -euo pipefail
+VAR_DIR='/var/lib/tensorleap/standalone'
+SCRIPT_LOG=$VAR_DIR/log.log
+
+function log(){
+  echo "[$(date)] [INFO]  $1" >> $SCRIPT_LOG
```

Comment:
Maybe you should also note the `INSTALL_ID` which is just random number generated each time you run the script and it also appears in the notifications

Reply:
Im logging it in line 44
```
log "INSTALL_ID = $INSTALL_ID"
```
Do you want the INSTALL_ID to be printed on each echo?